src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -47,7 +48,30 @@ public PersonCard(Person person, int displayedIndex) {
         phone.setText(person.getPhone().value);
         address.setText(person.getAddress().value);
         email.setText(person.getEmail().value);
-        person.getTags().forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        //person.getTags().forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        initTags(person);
+    }
+
+    /**
+     * Add comment!!
```

Comment:
Yes, please add properly header comments. Saying "Add comment!!" doesn't make it a proper header comment...